stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -206,8 +206,9 @@ internal class ChatDomainImpl internal constructor(
     }
 
     internal lateinit var repos: RepositoryHelper
-    private var syncState: SyncStateEntity? = null
-    internal lateinit var initJob: Deferred<SyncStateEntity?>
+    @Volatile
+    private var syncStateFlow: MutableStateFlow<SyncState?> = MutableStateFlow(null)
```

Comment:
```suggestion
    private val syncStateFlow: MutableStateFlow<SyncState?> = MutableStateFlow(null)
```

Reply:
Amazing, this got outdated between the time I drafted the comment and published the review, but GitHub opted not to remove it 😄